python/tests/test_table.py

```diff
@@ -657,3 +659,35 @@ def test_count_rows(db):
     assert len(table) == 2
     assert table.count_rows() == 2
     assert table.count_rows(filter="text='bar'") == 1
+
+
+def test_consistency(tmp_path):
+    db = lancedb.connect(tmp_path)
+    table = LanceTable.create(db, "my_table", data=[{"id": 0}])
+    # Note: we assert each of these have versions, in part to force them to do
+    # their lazy loading of the dataset
+    table_clone = db.open_table("my_table")
+    assert table_clone.version == table.version
+    table_consistent = db.open_table("my_table", read_consistency_interval=timedelta(0))
+    assert table_consistent.version == table.version
+    table_eventual = db.open_table(
+        "my_table", read_consistency_interval=timedelta(seconds=0.1)
+    )
+    assert table_eventual.version == table.version
+
+    table.add([{"id": 1}])
+    assert table_clone.version == table.version - 1
+    assert table_consistent.version == table.version
+    # assert table_eventual.version == table.version - 1
```

Comment:
Remove?

Reply:
Ah no that should be brought back. It was temporarily commented out while I was adding breakpoints.